### ISSUE TITLE
Wrong response to cwd command

### ISSUE THREAD

**Issue by igor-savin-ht** [BriteVerify/node-sftp-server#49]
FileZilla by default sends 'pwd' command and node-sftp-server is sending 'Current directory is: "."' response that FileZilla doesn't understand and fails with 'Failed to parse returned path.'.

Would be really helpful if we could override the logic for forming response to pwd command.

**Comment by igor-savin-ht** [comment]
Finally figured this out - apparently, 'realpath' listener is responsible for determining path for the pwd response. Might be worth mentioning that in readme :)

**Comment by uberbrady** [comment]
Yes, would you be able to pull together a PR documenting that? I'll merge it in as soon as possible.

**Comment by igor-savin-ht** [comment]
@uberbrady Sure, I'll give it a shot this week.

**Comment by msjoshi** [comment]
Does anyone have a sample `realpath` implementation? I added this implementation but Filezilla errors out 'server returns empty path'.
Below is my implementation. Is it the right implementation?

```
session.on('realpath', (path, callback) => {
			const name = `${path}/${config.sftp.tmpDirectory}/text.txt`;
			log.info(`Realpath is ${path}/${config.sftp.tmpDirectory}`);
			return callback(name);
		});
```
